internal/formats/spdx22json/model/element.go

```diff
@@ -3,7 +3,7 @@ package model
 type Element struct {
 	SPDXID string `json:"SPDXID"`
 	// Identify name of this SpdxElement.
-	Name string `json:"name"`
+	Name string `json:"name,omitempty"`
```

Comment:
Not sure if this will have other adverse effects since it is used by things such as [spdx22json.Package](https://github.com/anchore/syft/blob/main/internal/formats/spdx22json/model/package.go#L4) [.Item.Element](https://github.com/anchore/syft/blob/dfefd2ea4e9d44187b4f861bc202970247dd34c8/internal/formats/spdx22json/model/item.go#L4) but not allowed in [spdx22json.File](https://github.com/anchore/syft/blob/dfefd2ea4e9d44187b4f861bc202970247dd34c8/internal/formats/spdx22json/model/file.go#L20-L30), something like this is needed.